File: app/src/main/java/com/chatserver/dd/chat/MainActivity.java
package com.chatserver.dd.chat;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

import com.chatserver.dd.chat.Activities.ActivityAllWords;
import com.chatserver.dd.chat.Activities.ActivityServerWords;
import com.chatserver.dd.chat.Activities.ActivityVipWords;

import static com.chatserver.dd.chat.R.layout.activity_main;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {


    //Buttons on MainAcitivity
    private Button mBtn1;
    private Button mBtn2;
    private Button mBtn3;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(activity_main);
        initView();


    }

    private void initView() {
        mBtn1 = (Button) findViewById(R.id.btn1);
        mBtn1.setOnClickListener(this);
        mBtn2 = (Button) findViewById(R.id.btn2);
        mBtn2.setOnClickListener(this);
        mBtn3 = (Button) findViewById(R.id.btn3);
        mBtn3.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        Bundle bundle = new Bundle();
        Intent intent1 = null;
        switch (v.getId()) {
            case R.id.btn1:

                //open new Activity
                intent1 = new Intent(this, ActivityAllWords.class);
                break;

            case R.id.btn2:

                //open new Activity
                intent1 = new Intent(this, ActivityServerWords.class);
                break;

            case R.id.btn3:// TODO 18/05/31

                //open new Activity
                intent1 = new Intent(this, ActivityVipWords.class);
                break;
        }
        startActivity(intent1);
    }
}

File: app/src/main/java/com/chatserver/dd/chat/main/MainActivity.kt
package com.chatserver.dd.chat.main

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.chatserver.dd.chat.R
import com.chatserver.dd.chat.blocks.admin.ActivityAdmin
import com.chatserver.dd.chat.blocks.chat.ActivityChat
import com.chatserver.dd.chat.blocks.players.ActivityPlayers
import com.chatserver.dd.chat.blocks.server.ActivityServer
import com.chatserver.dd.chat.blocks.vip.ActivityVip
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initView()


    }

    private fun initView() {
        btn_AllWords.setOnClickListener(this)
        btn_Server.setOnClickListener(this)
        btn_Vip.setOnClickListener(this)
        btn_Admins.setOnClickListener(this)
        btn_Cmd.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        val intent: Intent?
        when (v.id) {
            R.id.btn_AllWords -> {
                // TODO 18/09/06
                intent = Intent(applicationContext, ActivityChat::class.java)
                startActivity(intent)
            }
            R.id.btn_Server -> {
                // TODO 18/09/06
                intent = Intent(applicationContext, ActivityServer::class.java)
                startActivity(intent)
            }
            R.id.btn_Vip -> {
                // TODO 18/09/06
                intent = Intent(applicationContext, ActivityVip::class.java)
                startActivity(intent)
            }
            R.id.btn_Admins -> {
                // TODO 18/09/06
                intent = Intent(applicationContext, ActivityAdmin::class.java)
                startActivity(intent)
            }
            R.id.btn_Cmd -> {
                intent = Intent(applicationContext, ActivityPlayers::class.java)
                startActivity(intent)
            }
            else -> {
            }
        }
    }
}